src/test/java/api/loans/CheckOutByBarcodeTests.java

```diff
@@ -55,6 +47,15 @@
 import api.support.http.InventoryItemResource;
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Seconds;
+import org.junit.Test;
+
+import org.folio.circulation.domain.policy.Period;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseHandler;
 
 public class CheckOutByBarcodeTests extends APITests {
```

Comment:
I think there needs to be a test where the same loan policy is used in different rules that would result in different counts of items (as described in the example on the JIRA issue)